src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -10,15 +10,70 @@ import { useSmallScreen } from "context/useSmallScreen";
 import {
   ConfirmationButton,
   Icon,
+  Tooltip,
   useNotify,
 } from "@canonical/react-components";
 import classnames from "classnames";
 import { useToastNotification } from "context/toastNotificationProvider";
+import { filterUsedByType } from "util/usedBy";
+import { ResourceType } from "util/resourceDetails";
 
 interface Props {
   project: LxdProject;
 }
 
+const generateProjectUsedByTooltip = (project: LxdProject) => {
+  const resourceLabelAndLink: Record<string, { label: string; link: string }> =
+    {
+      instance: {
+        label: "Instances",
+        link: `/ui/project/${project.name}/instances`,
+      },
+      profile: {
+        label: "Profiles",
+        link: `/ui/project/${project.name}/profiles`,
+      },
+      image: {
+        label: "Images",
+        link: `/ui/project/${project.name}/images`,
+      },
+      volume: {
+        label: "Custom volumes",
+        link: `/ui/project/${project.name}/storage/volumes`,
+      },
+    };
+
+  const resourceTypes = Object.keys(resourceLabelAndLink);
+  const usedByItems: ReactNode[] = [];
+  for (const resourceType of resourceTypes) {
+    const usedBy = filterUsedByType(
+      resourceType as ResourceType,
+      project.used_by,
+    );
+
+    if (usedBy.length > 0) {
+      const label = resourceLabelAndLink[resourceType].label;
+      const link = resourceLabelAndLink[resourceType].link;
+      usedByItems.push(
+        <li
+          key={resourceType}
+          className="p-list__item is-dark u-no-margin--bottom"
+        >
+          {link ? <Link to={link}>{label}</Link> : label} ({usedBy.length})
```

Comment:
```suggestion
          {<Link to={link}>{label}</Link>} ({usedBy.length})
```
I think link is never empty, when label is set, isn't it?